controllers/controllers.news.js

```diff
@@ -0,0 +1,9 @@
+const { fetchTopics } = require("../models/models.news");
+
+
+exports.getTopics = (req, res, next) => {
+    fetchTopics().then(( result) => {
+        console.log(result);
+        res.status(200).send({ topics:result });
```

Comment:
you can rename result in the .then block topics directly and please remove any unnecessary console.log